climpred/preprocessing/shared.py

```diff
@@ -0,0 +1,109 @@
+import numpy as np
+import xarray as xr
+
+from ..constants import CLIMPRED_ENSEMBLE_DIMS
+from .mpi import get_path
+
+
+def set_integer_axis(ds, lead_offset=1, time_dim='time'):
+    """CMIP6 DCPP preprocessing before the aggreatations of intake-esm happen."""
+    # set time_dim to integers starting at lead_offset
+    ds[time_dim] = np.arange(lead_offset, lead_offset + ds[time_dim].size)
+    return ds
+
+
+def load_hindcast(
```

Comment:
Isn't this a replica of `load_hindcast` from the `mpi.py`? I think the only thing that belongs in `mpi.py` right now is `get_path` since it's specific to mistral and expects MPI-style output naming

Reply:
Only get_path is mpi specific. There this goes into shared

Reply:
Correct. And right now there are two duplicate functions in `mpi` that can be deleted I think.